K8s-deployment/Charts/resource-server/example-azure-resource-values.yaml

```diff
@@ -80,7 +80,7 @@ databroker:
 encryption:
   enabled: true
   nodeSelector:
-    node.kubernetes.io/instance-type: t3.medium
+    node.kubernetes.io/instance-type: t3a.medium
```

Comment:
The nodeSelector should be node.kubernetes.io/instance-type: Standard_B2s
for azure